src/structure_managers/adaptor_neighbour_list.hh

```diff
@@ -780,6 +790,9 @@ namespace rascal {
      */
     size_t n_ghosts;
 
+    //! number of atoms in the unit cell
```

Comment:
maybe add: "and considered as centers?"

Reply:
this is true from n_centers but not for n_atoms